akkeris-service-token-generator/README.md

```diff
@@ -0,0 +1,23 @@
+# Akkeris Service Token Generator
+
+## Instructions
+
+Set your `kubectl` context to the cluster that your Akkeris installation is running in.
+
+Run `source get_config.sh` to save the JWT private & public keys needed to generate a token as environment variables.
+
+Set the following additional environment variables: 
+ 
+ - `TOKEN_TTL`: Time that the token should be valid for (in seconds). Default is 10 years (315360000) for a long-lived service token
+ - `USERNAME`: Name of the service requesting the service token
+ - `AKKERIS_API`: Endpoint of the Akkeris API (e.g. https://apps.yourdomain.io)
```

Comment:
Why require these as env vars instead of args and pull them from `process.argv `? That would seem simpler and cleaner to me. Same question for those below.

Reply:
Mainly because validating args is a huge pain (there are entire libraries dedicated to it) and I didn't want to take very much time on this haha